modules/nodes-cluster-overcommit-project-disable.adoc

```diff
@@ -7,29 +7,61 @@
 [id="nodes-cluster-overcommit-project-disable_{context}"]
 = Disabling overcommitment for a project
 
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 When enabled, overcommitment can be disabled per-project. For example, you can allow infrastructure components to be configured independently of overcommitment.
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 
-.Procedure
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+If required by your use case, you can disable overcommitment on any project that is not managed by Red Hat. For a list of projects that cannot be modified, see _Red Hat Managed resources_ in _Additional resources_.
 
-To disable overcommitment in a project:
+.Prerequisites
+* You are logged in to the cluster using an account with cluster administrator or cluster editor permissions.
```

Comment:
```suggestion
* You are logged in to the cluster by using an account with cluster administrator or cluster editor permissions.
```

Reply:
I'm vetoing this one - "by using" implies that the account that has those permissions is in some way responsible for the logging in action. It is not. I can use "You are logged in to the cluster as an account with..." or "You are logged in to the cluster with an account that has ..." instead, but "by using" would be odd here.